### PR TITLE
Bug 1994696: Clarify not eligible disk error message

### PR DESCRIPTION
**Pull request by michaellevy101**
Currently we say "Require a disk of at least 120 GB" which makes it seem that
the capacity isn't large enough, even if the real reason that disks are insufficient
aren't related to capacity.

# Assisted Pull Request

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/host/transition_test.go [hunk 1]
```diff
@@ -2917,7 +2917,7 @@ var _ = Describe("Refresh Host", func() {
 				statusInfoChecker: makeValueChecker(formatStatusInfoFailedValidation(statusInfoInsufficientHardware,
 					"Machine Network CIDR is undefined; the Machine Network CIDR can be defined by setting either the API or Ingress virtual IPs",
 					"The host is not eligible to participate in Openshift Cluster because the minimum required RAM for any role is 8.00 GiB, found only 130 bytes",
-					"Require a disk of at least 120 GB",
+					"At least one eligible disk is required, please specific disks for non-eligibility reasons",
```
Comment (avishayt) [inline internal/host/transition_test.go:2920]:
```suggestion
					"No eligible disks were found, please check specific disks to see why they are not eligible",
```

**Review by avishayt — APPROVED** [review]

**Comment by flaper87** [comment]
@michaellevy101 could you please shorten the PR title and add a description to it?

**Pull request merged**